<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Basic_Info</class>
 <widget class="QWidget" name="Basic_Info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>522</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>思源黑体 CN Medium</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="8">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>请按说明填写，填写完成后请点击“写入”，然后关闭此窗口，在主窗口的工具栏内找到“更新”并点击其中的“刷新基本信息”选项卡</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>NPC名：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QLineEdit" name="name_edit"/>
     </item>
     <item row="1" column="5" colspan="3">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>（填写NPC头顶的白色部分）</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>称号：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="4">
      <widget class="QLineEdit" name="nickname_edit"/>
     </item>
     <item row="2" column="5" colspan="3">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>（填写NPC头顶的蓝色部分中&lt;&gt;内的（不包括&lt;&gt;）没有的请写“无”）</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>性别：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="QComboBox" name="gender_edit">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>未知</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>男</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>女</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="5" colspan="3">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>（性别未知的请填“未知”）</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>职业：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="4">
      <widget class="QLineEdit" name="job_edit"/>
     </item>
     <item row="4" column="5" colspan="3">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>（根据实际情况，售卖东西请写“商人”，做菜请写“厨师”，以此类推，不知道请写“未知”）</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>位置：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="4">
      <widget class="QLineEdit" name="location_edit"/>
     </item>
     <item row="5" column="5" colspan="3">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>填写NPC所在的地理位置，使用地名，地名之间使用“·”连接，尽量不用方位词。
填写示例：璃月港·吃虎岩</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>所属国家：</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="3">
      <widget class="QComboBox" name="country_edit">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>未知</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>蒙德</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>璃月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>稻妻</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>须弥</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>枫丹</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>纳塔</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>至冬</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="5" colspan="3">
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>（所属国家未知的请填“未知”）</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>对话赠礼：</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QCheckBox" name="gift_exist">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>存在</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QLabel" name="gift_name_label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>赠礼：</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4" colspan="2">
      <widget class="QLineEdit" name="gift_name_edit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="6">
      <widget class="QLabel" name="gift_num_label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>数量：</string>
       </property>
      </widget>
     </item>
     <item row="7" column="7">
      <widget class="QLineEdit" name="gift_num_edit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>实装版本：</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2" colspan="3">
      <widget class="QComboBox" name="version_edit">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>未知</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2.0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2.1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2.2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2.3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2.4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2.6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2.7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2.8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2.9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3.0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3.1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3.2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3.3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3.4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3.6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3.7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3.8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3.9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4.0</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="5" colspan="3">
      <widget class="QLabel" name="label_18">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>（未知请填写“未知”）</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>所属组织：</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2" colspan="3">
      <widget class="QLineEdit" name="org_edit"/>
     </item>
     <item row="9" column="5" colspan="3">
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>（可填写工作单位，没有或未知的请填写“未知”）</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QLabel" name="label_22">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>相关系统：</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2" colspan="3">
      <widget class="QComboBox" name="system_edit">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>无</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>委托</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>声望</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>活动</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>商店</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>烹饪</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>尘歌壶</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="10" column="5" colspan="3">
      <widget class="QLabel" name="label_21">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>（可以在生成后手动更改）</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="8">
      <widget class="QPushButton" name="update_basic_info">
       <property name="font">
        <font>
         <family>思源黑体 CN Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>更新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
